packages/pigment-css-react/tests/useTheme/fixtures/useTheme.input.js

```diff
@@ -1,4 +1,6 @@
-import { useTheme } from '../zero-styled';
+import { useTheme } from '@pigment-css/react';
+
+console.log(useTheme);
```

Comment:
```suggestion
```

Reply:
Is the console.log intentional?

Reply:
This is part of the test to make sure we don't have the same issue that this PR fixes.

Reply:
I see, got it. Can you add a comment that it's intentional? I am sure others will ask if they see console.log